### ISSUE TITLE
Please compile a recent version of BIMP for Windows

### ISSUE THREAD

**Issue by wrack2** [alessandrofrancesconi/gimp-plugin-bimp#359]
Please compile a recent version of BIMP for Windows.
The last release v2.6 is almost 2 years old.
New features, such as AVIF support, have been added.
Thank you.

**Comment by Nickonimus** [comment]
Yes please, avif support must-have for now, since it twice smaller than jpg and already supported by all browsers!

**Comment by Rufus13apostol** [comment]
And for linux!

**Comment by fiyakamstore** [comment]
At the moment I want to remove a batch photo background with bimple using gmic. but everything got messy .
